loader/base_loader.py

```diff
@@ -108,6 +118,106 @@ def __init__(self, dss_endpoint: str, staging_bucket: str, google_project_id: st
         dss_config['DSSClient'].swagger_url = f'{self.dss_endpoint}/swagger.json'
         self.dss_client = DSSClient(config=dss_config)
 
+    @staticmethod
+    def get_s3_metadata_client(aws_meta_cred, session='NIH-Test', duration=43199):
+        """
+        Access AWS credentials from a file and supply a client for them.
+
+        :param aws_meta_cred: File containing an AWS ARN for an AssumedRole, e.g.:
+                              'arn:aws:iam::************:role/ROLE_NAME_HERE'
+        :param duration: How long, in seconds, the AssumedRole will be valid for.
+        :return: An AWS s3 client object authorized with the above credentials or None.
+        """
+        if not aws_meta_cred:
+            return None
+
+        client = boto3.client('sts')
```

Comment:
`sts_client` would be a better name (there are a lot of clients in this program!)